airbyte-cdk/bulk/toolkits/load-http/src/main/kotlin/io/airbyte/cdk/load/http/okhttp/AirbyteOkHttpClient.kt

```diff
@@ -37,8 +37,10 @@ class AirbyteOkHttpClient(
                 .method(request.method.toString(), request.body?.toRequestBody())
                 .apply { request.headers.forEach { header -> addHeader(header.key, header.value) } }
                 .build()
-        val response: okhttp3.Response =
-            FailsafeCall.with(retryPolicies[0], *retryPolicies.drop(1).toTypedArray()).compose(client.newCall(okhttpRequest)).execute()
+        val response = when (retryPolicies.isEmpty()) {
+            true -> client.newCall(okhttpRequest).execute()
+            false -> FailsafeCall.with(retryPolicies[0], *retryPolicies.drop(1).toTypedArray()).compose(client.newCall(okhttpRequest)).execute()
```

Comment:
actually, why do you need to manually extra the first element?
I thought `FailsafeCall.with(*retryPolicies.toTypedArray())` would do the same.